### DISCUSSION TITLE
Update to Brazilian Portuguese locale [br-pt]

### DISCUSSION THREAD

**Discussion by ebclaude** [aclap-dev/video-downloadhelper#337] [Language & Translation]
[messages.json](https://github.com/aclap-dev/video-downloadhelper/files/14784094/messages.json)

**Comment by paulrouget** [comment]
Thank you! I will add that to the upcoming version.

**Comment by paulrouget** [comment]
Pardon my ignorance, but how is it different from https://github.com/aclap-dev/video-downloadhelper/discussions/352 ?

**Comment by MarcoBrandao** [comment]
Hi.
In the file above, this line "mup_max_variants_help": "Cada vídeo vem em formatos diferentes. Mostramos primeiro a melhor versão e também alguns formatos alternativos (vaiantes).", has a wrong word "(vaiantes)". It should be the word "(variantes)".
I don't have the original file to check, but vaiantes is to a valid word in Brazilian Portuguese.
Other phrases and words in the file are correct.

**Comment by ericsantos1** [comment]
[pt_BR.txt](https://github.com/aclap-dev/video-downloadhelper/files/15410182/pt_BR.txt)
Translation for Video DownloadHelper Version 9

**Comment by carloslischka** [comment]
Sending some translations. Thanks.
[pt_BR.txt](https://github.com/aclap-dev/video-downloadhelper/files/15499319/pt_BR.txt)

**Comment by paulrouget** [comment]
These will all be adde to v9.1 Thanks!
